examples/src/main/resources/api/testApi.yaml

```diff
@@ -0,0 +1,193 @@
+openapi: "3.0.0"
+
+info:
+  version: "0.1"
+  title: Scorex Node API example
+  description: API docs for example Scorex project.
+  contact:
+    name: Scorex framework Team
+    email: kushi@protonmail.com
+    url: https://github.com/scorexfoundation/scorex
+  license:
+    name: CC0 1.0 Universal
+    url: https://raw.githubusercontent.com/ScorexFoundation/Scorex/master/COPYING
+
+components:
+  schemas:
+    # Objects
+    ModifierId:
+      description: Base58-encoded 32 byte modifier id
+      type: object
+      required:
+        - modifierId
+      properties:
+        modifierId:
+          type: string
+          example: D2bXMwWN8P9nWJ9qqwZJLauAdcZHX9n6s91QQ9vK6Zu4
+
+    Digest32:
+      description: Base58-encoded 32 byte digest
+      type: object
+      required:
+        - digest
+      properties:
+        digest:
+          type: string
+          example: FjX5cPuwMc2ocDLPWzt6jq29BXjvU6d5w5XURDJ6dmoM
+    Peer:
+      type: object
+      required:
+        - address
+      properties:
+        address:
+          type: string
+          example: 127.0.0.1:5673
+        name:
+          type: string
+          example: mynode
+        nonce:
+          type: integer
+          # int32
+          format: int64
+          example: 123456
+        lastSeen:
+          type: integer
+          # int32
+          format: int64
+          example: 123456
+
+
+paths:
+
+  /peers/all:
+    get:
+      summary: Get all known peers
+      operationId: getAllPeers
+      tags:
+        - peers
+      responses:
+        200:
+          description: Array of peer objects
+          content:
+            application/json:
+              schema:
+                type: array
+                items:
+                  $ref: '#/components/schemas/Peer'
+
+  /peers/connected:
+    get:
+      summary: Get current connected peers
+      operationId: getConnectedPeers
+      tags:
+        - peers
+      responses:
+        200:
+          description: Array of peer objects
+          content:
+            application/json:
+              schema:
+                type: array
+                items:
+                  $ref: '#/components/schemas/Peer'
+
+  /peers/connect:
+    post:
+      summary: Get current size of the unconfirmed transactions pool
```

Comment:
Is this the correct summary for this path?

Reply:
@ceilican I think not. I fixed this one here https://github.com/ergoplatform/ergo/pull/108/files#diff-f11bc37099101a6156775fbde1871902R712

Reply:
Your fix seems good to me.

Reply:
@ceilican But it was done in ergo repository =)

Reply:
Yes, I know. I think it should be done here (in Scorex, in this PR) too.

And I wonder if it would be possible/desirable, in the future (in another PR), to reduce code and documentation duplication at the api level between scorex('s example) and ergo.  It seems to me that right now duplication is only avoided at the network layer. All other layers remain completely abstract in Scorex's core and need to be (re-)implemented, possibly in similar ways, by every blockchain system built with Scorex. Perhaps, in the future, Scorex could provide some standard api routes/paths already pre-implemented.

@daron666, is ergo's api significantly different from the api of scorex's example?

@kushti, should we include the api into the scope of that architecture review that we have discussed?

Reply:
@ceilican Ergo's api is a superset over scorex api.